.github/workflows/drivex.yml

```diff
@@ -56,6 +55,13 @@ jobs:
         echo "PANDO_TEST_DISCOVERY_TIMEOUT=600" >> $GITHUB_ENV
         echo "IMAGE_VERSION=$(git log --pretty="%h" -1 Dockerfile.dev)" >> $GITHUB_ENV
         echo "PANDO_CONTAINER_ENV=-e=PANDORT_TESTS=ON" >> $GITHUB_ENV
+        echo ${{ runner.name }}
+        if [ ${{ runner.name }} == 'zerberus-0' ]; then
```

Comment:
I thought there were 4 runners?

Reply:
Also does zerberus have hyperthreads enabled?

Reply:
no right now we are doing 2.

Reply:
yes hyperthreads are enabled, but these splice the sockets